scripts/calendar/hebrew_generator.py

```
@@ -44,49 +44,134 @@
+
+
+class _Lunisolar:
+    """
+    This class generates Gregorian dates for Hebrew calendar based holidays.
+
+    References:
+        * <https://en.wikipedia.org/wiki/Hebrew_calendar>
+    """
+
+    @staticmethod
+    def leap(year):
```

Ruff (0.15.9)

[warning] Missing return type annotation for staticmethod `leap`

(ANN205)